src/System.Management.Automation/commands/utility/FormatAndOutput/common/DisplayDatabase/displayDescriptionData.cs

```diff
@@ -713,6 +718,11 @@ public abstract class PSControl
         /// </summary>
         public bool OutOfBand { get; set; }
 
+        /// <summary>
+        /// Set explicit limit for Width. Default width is a console host width.
+        /// </summary>
+        public int? Width { get; set; }
+
```

Comment:
I believe @JamesWTruher was working on remoting interop tests - hopefully we'll get those at some point.